spec/features/trainees/submit_for_trn_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "rails_helper"
 
-feature "submit for TRN" do
+xfeature "submit for TRN" do
```

Comment:
Any reason why we're ignoring this?

Reply:
Similar to above, although for this one it might be easier to get working - not sure if it's worth it while the other is ignored?

Reply:
It's working now, with a bit of refactoring